src/main/java/picocli/CommandLine.java

```diff
@@ -11581,6 +11583,21 @@ private static String extractName(ResourceBundle rb) {
                     return (String) ResourceBundle.class.getDeclaredMethod("getBaseBundleName").invoke(rb);
                 } catch (Exception ignored) { return "?"; }
             }
+            public Messages parent() {
+            	CommandSpec parentSpec = this.spec.parent();
+            	if ( parent==null || parent.spec!=parentSpec ) {
```

Comment:
Can we keep the same style as the existing code?
Specifically, no spaces after opening brace `(` or before closing brace `)`, but we do have spaces before and after comparison like `==` and `!=`.

So, for example, I would like the above line to look like this:
```java
if (parent == null || parent.spec != parentSpec) {
```
Similarly for the other `if`, `while` and `return` statements below.

Reply:
Hey @remkop, the whitespacing has been updated per your instructions. Just as a heads up, all builds/jobs for my branch completed successfully on my end, but for some reason they failed when running on your repo. I don't know why, but sometimes I have to run the workflow multiple times for all jobs to complete successfully. 

I guess that build workers/runners on GitHub has issues sometimes with name resolution when trying to download packages?

Reply:
About the CI builds/jobs failing on GitHub, I also noticed and I don't consider it a problem because like you said, these jobs seems to fail because runners fail to download dependencies sometimes.

Reply:
I just addressed the other whiteshacing issues from your later comment. I'll mark this one as completed as well. I just pushed so my branch is currently running the workflow.